core/src/main/scala/blueprint/Render.scala

```diff
@@ -0,0 +1,127 @@
+package nelson
+package blueprint
+
+import nelson.Datacenter.StackName
+import nelson.Manifest._
+import nelson.blueprint.EnvValue._
+import nelson.docker.Docker.Image
+
+object Render {
```

Comment:
This file should be well documented as it's the core of the blueprint templaing engine

Reply:
Yep, will follow up all the docs and tests in future PR as we discussed.